plugins/plugin-chart-echarts/README.md

```diff
@@ -0,0 +1,40 @@
+## @superset-ui/plugin-chart-echarts-line
+
+[![Version](https://img.shields.io/npm/v/@superset-ui/plugin-chart-echarts.svg?style=flat-square)](https://img.shields.io/npm/v/@superset-ui/plugin-chart-echarts.svg?style=flat-square)
+[![David (path)](https://img.shields.io/david/apache-superset/superset-ui.svg?path=packages%2Fsuperset-ui-plugin-chart-echarts&style=flat-square)](https://david-dm.org/apache-superset/superset-ui?path=packages/superset-ui-plugin-chart-echarts)
+
+This plugin provides Echarts viz plugins for Superset:
+- Timeseries Chart (combined line, area bar with support for predictive analytics)
+- Pie Chart
+
+### Usage
+
+Configure `key`, which can be any `string`, and register the plugin. This `key` will be used to lookup this chart throughout the app.
+
+```js
+import {
+  EchartsTimeseriesChartPlugin,
+  EchartsPieChartPlugin,
+} from '@superset-ui/plugin-chart-echarts';
+
+new EchartsTimeseriesChartPlugin()
+  .configure({ key: 'echarts-ts' })
+  .register();
+new EchartsPieChartPlugin()
+  .configure({ key: 'echarts-pie' })
+  .register();
+```
+
+Then use it via `SuperChart`. See [storybook](https://apache-superset.github.io/superset-ui/?selectedKind=plugin-chart-echarts-line) for more details.
```

Comment:
I don't _think_ this story exists. We should either remove this line, or add the story.

Reply:
Story has now been added with all applicable knobs (check screenshot)